Library/Formula/ffmpeg.rb

```diff
@@ -64,6 +65,7 @@ class Ffmpeg < Formula
   depends_on "libvidstab" => :optional
   depends_on "x265" => :optional
   depends_on "openssl" => :optional
+  depends_on "webp" if build.with? "libwebp"
```

Comment:
Can just do `depends_on "webp" => :optional` and name the option `with-webp` instead.